jazz_events/index.js

```diff
@@ -40,382 +40,449 @@ module.exports.handler = (event, context, cb) => {
 
 	try {
 		//GET Handler
-		if (event !== undefined && event.method !== undefined && event.method === 'GET') {
-			async.series({
-				get_events: function (callback) {
-					var filter = "";
-					var attributeValues = {};
+		if (event && event.method && event.method === 'GET') {
+			getEvents(event, config, dynamodb)
+			.then((result) => mapGetEventData(result, event))
+			.then(function(result){
```

Comment:
Please be consistent in usage of arrow functions compared to line 45.

Reply:
Well  i am seeing this in other places as well which you can update

Reply:
Updated